examples/logical_device_context/main.tf

```diff
@@ -13,12 +13,17 @@ provider "aci" {
   insecure = true
 }
 
+resource "aci_tenant" "example" {
+  name = "test_acc_tenant"
+}
+
 resource "aci_logical_device_context" "example" {
-  tenant_dn         = aci_tenant.tenentcheck.id
-  ctrct_name_or_lbl = "default"
-  graph_name_or_lbl = "any"
-  node_name_or_lbl  = "N1"
-  context           = "ctx1"
-  description       = "from terraform"
-  name_alias        = "example"
+  tenant_dn                          = aci_tenant.example.id
+  ctrct_name_or_lbl                  = "default"
+  graph_name_or_lbl                  = "any"
+  node_name_or_lbl                   = "N1"
+  context                            = "ctx1"
+  description                        = "from terraform"
+  name_alias                         = "example"
+  relation_vns_rs_l_dev_ctx_to_l_dev = "uni/tn-MyTenant01/lDevVip-LoadBalancer01"
```

Comment:
as your tenant above is not called MyTenant01 you might want to use test_acc_tenant here for the tenant name

Reply:
Yeah missed it